docs/pages/access-controls/idps/saml-grafana.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Authenticate to Grafana with Teleport
+description: Configure Grafana to use identities provided by Teleport.
+---
+
+Grafana is an open-source observability platform. Their enterprise version supports
+SAML authentication. This guide will help you configure Teleport as a SAML provider,
+and Grafana to accept the identities it provides.
+
+Note that Teleport can act as an identity provider to any SAML-compatible service,
+not just those running behind the Teleport App Service.
+
+## Prerequisites
+
+- An instance of Grafana Enterprise, with edit access to `grafana.ini`.
+  - A certificate solution to create TLS certificates/keys for the SAML connection.
```

Comment:
Can we be more precise here? A quick google of _certificate solution_ pulls up a ton of unrelated items.

Reply:
Updated.